doc/adr/0004-add-indexes-to-the-mongodb-image.md

```diff
@@ -0,0 +1,21 @@
+# 4. Add Indexes to the MongoDB image
+
+Date: 2017-03-21
+
+## Status
+
+Accepted
+
+## Context
+
+A spike has been done to check if the image can be layered, i.e. have the base image and then add the index as another layer, so
```

Comment:
If the record is changed as suggested the context becomes about not wanting to spend effort on creating a bespoke image for `gp-finder`.